fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1501,6 +1513,17 @@ public boolean isCboPredicateSubfieldPath() {
     @VarAttr(name = ENABLE_PLAN_SERIALIZE_CONCURRENTLY)
     private boolean enablePlanSerializeConcurrently = true;
 
+    @VarAttr(name = FOLLOWER_QUERY_FORWARD_MODE, flag = VariableMgr.INVISIBLE | VariableMgr.DISABLE_FORWARD_TO_LEADER)
```

Comment:
I found that this session var can be a little bit confision, because for a query sent to a follower, it either runs on the follower itself or is forwarded to leader, so do you think is it better to use a boolean var instead of a string var, like `FOLLOWER_QUERY_FORWARD_TO_LEADER = true|false`

Reply:
1. in this pr, once `follower_query_forward_mode ` is set(leader/follower), follower's queries can always proxied to leader/follower.
2. `follower_query_forward_mode = true|false` cannot represent this meaning, because we cannot distinguish the default state which is not set.

I added some comments below:
```
    // Flag to control whether to proxy follower's query statement to leader/follower.
    public enum FollowerQueryForwardMode {
        DEFAULT,    // proxy queries by the follower's replay progress (default)
        FOLLOWER,   // proxy queries to follower no matter the follower's replay progress
        LEADER      // proxy queries to leader no matter the follower's replay progress
    }
```